### PR TITLE
Add JetBrains Toolbox

### PR DESCRIPTION
**Pull request by MartinX3**
Closed https://github.com/JonasGroeger/jetbrains-ppa/issues/10

---
This change is [Reviewable](https://reviewable.io/reviews/jonasgroeger/jetbrains-ppa/12)

### CODE REVIEW

File: packages/jetbrains-toolbox-app/debian/postrm [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+set -e
+
+rm -f /opt/jetbrains-toolbox-app.tar.gz # Remove the empty file, if the download failed.
+rm -rf /opt/jetbrains-toolbox-app
+rm -rf ~/.local/share/JetBrains # It installs itself on first boot in this folder.
```
Comment (JonasGroeger) [inline packages/jetbrains-toolbox-app/debian/postrm]: According to https://toolbox-support.jetbrains.com/hc/en-us/articles/115000978804-Where-are-the-IDEs-located-on-my-hard-drive- this is also where the Toolbox App puts the IDEs. Do we really want to remove all IDEs on toolbox installation?

**Comment by JonasGroeger** [comment]
Please remove the `packages/jetbrains-toolbox-app/jetbrains-toolbox-app.xcf` file and squash all commits into one.

File: packages/jetbrains-toolbox-app/debian/postrm (hunk 1 above)
Comment (MartinX3) [reply packages/jetbrains-toolbox-app/debian/postrm]: I assumed, that, when you you want to remove the toolbox, you also want to remove every IDE.

But that could be the wrong way.
I switch it to a Hint, that the User **should** remove this folder by hand, if he wants to remove the IDE's.

**Comment by MartinX3** [comment]
I don't find a way to squash all commits into one in this alreasy opened PR.
I will close this and open a new one, to do this.

**Pull request closed without merging**